src/routes.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+import Offcanvas from 'react-bootstrap/Offcanvas';
+import Button from 'react-bootstrap/Button';
+
+function Root() {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      <Navbar bg="success" expand="lg">
+        <Navbar.Toggle aria-controls="offcanvasNavbar" />
+        <Navbar.Brand href="#">Navbar Offcanvas</Navbar.Brand>
```

Comment:
Above the burger button I can see another button, it is the close button, but, shouldn't we see it until you expand the navbar?
When I click the 'burger' icon to see the navbar
![image](https://github.com/luigirazum/book-destinations/assets/67355479/a4c8f521-83cb-429a-ad44-af4ad7abf3e5)

The navbar title stays fixed.
![image](https://github.com/luigirazum/book-destinations/assets/67355479/12acb9cb-5077-4d6a-8466-76940a411f9d)

Would you mind making the close button hide and show according to the burger button was clicked.
And for the navbar title, wouldn't be better to align it to the top? as the navbar is expanded to take the 100vh.